Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -0,0 +1,880 @@
+            get : function() {
+                return showRequestVolumes();
+            },
+            set : function(value) {
+                showRequestVolumes(value);
+                if (that._tileset) {
+                    that._tileset.debugShowViewerRequestVolume  = value;
+                }
+            }
+        });
+        /**
+         * Gets or sets the flag to show request volumes.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.showRequestVolumes = false;
+
+        var showGeometricError = knockout.observable();
+        knockout.defineProperty(this, 'showGeometricError', {
+            get : function() {
+                return showGeometricError();
+            },
+            set : function(value) {
+                showGeometricError(value);
+                if (that._tileset) {
+                    that._tileset.debugShowGeometricError = value;
+                }
+            }
+        });
+        /**
+         * Gets or sets the flag to show tile geometric error.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.showGeometricError = false;
+
+        var freezeFrame = knockout.observable();
+        knockout.defineProperty(this, 'freezeFrame', {
+            get : function() {
+                return freezeFrame();
+            },
+            set : function(value) {
+                freezeFrame(value);
+                if (that._tileset) {
+                    that._tileset.debugFreezeFrame = value;
+                    that._scene.debugShowFrustumPlanes = value;
+                }
+            }
+        });
+        /**
+         * Gets or sets the flag to suspend updates.  This property is observable.
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.freezeFrame = false;
+
+        var maximumSSE = knockout.observable();
+        knockout.defineProperty(this, 'maximumSSE', {
+            get : function() {
+                return maximumSSE();
+            },
+            set : function(value) {
+                maximumSSE(value);
+                if (that._tileset) {
+                    that._tileset.maximumScreenSpaceError = value;
+                }
+            }
+        });
+        /**
+         * Gets or sets the maximum screen space error.  This property is observable.
+         *
+         * @type {Number}
+         * @default 16
+         */
+        this.maximumSSE = 16;
+
+        var dynamicSSEDensity = knockout.observable();
+        knockout.defineProperty(this, 'dynamicSSEDensity', {
+            get : function() {
+                return dynamicSSEDensity();
+            },
+            set : function(value) {
+                dynamicSSEDensity(value);
+                if (that._tileset) {
+                    that._tileset.dynamicScreenSpaceErrorDensity = Math.pow(value, 6);
```

Comment:
This is the only change I wasn't 100% sure on.  This value was previously being [set in the view](https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Widgets/CesiumInspector/Cesium3DTilesInspector.js#L335-L339)
Is there any significance to the [exponent of 6?](https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Widgets/CesiumInspector/Cesium3DTilesInspector.js#L127)

@austinEng